### PR TITLE
chore(ui): environment

### PR DESCRIPTION
**Pull request by cezres**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 /coverage
 /build
 
+.env
```
Comment (classicalliu) [inline .gitignore]: Why ignore `.env` and any env file used ?

Comment (cezres) [reply .gitignore]: `.env` is the default environment, `client_id` is in this file

Comment (classicalliu) [reply .gitignore]: Don't ignore this file, remove `client_id` or delete file if not used is just ok.

Comment (cezres) [reply .gitignore]: ok

**Pull request merged**